ci/run_easymesh_cert.sh

```diff
@@ -54,29 +56,47 @@ is_prplmesh_device() {
     return 1
 }
 
+deploy_dummy_prplmesh() {
+    info "Deploy prplmesh locally to /opt/prplMesh"
+    rm -rf /opt/prplMesh
```

Comment:
Is this script run as root?

Reply:
Yes, because run_test_file.sh needs to be run as root.